### PR TITLE
completed rolebinding step and relationship

### PR DESCRIPTION
**Pull request by poornima-metron**

### CODE REVIEW

File: src/steps/resource-manager/container-services/converters.ts [hunk 1]
```diff
@@ -23,6 +23,16 @@ export function getAccessRoleKey(name, location) {
   return `azure_access_role_${name}_location_${location} `;
 }
 
+function getEntityFromId(id: string, entityName): string {
+  const parts = id.split('/');
+  const index = parts.indexOf(entityName);
+  if (index !== -1 && index + 1 < parts.length) {
+    return parts[index + 1];
+  } else {
+    throw new Error('Invalid id format');
```
Comment (suraj-metron) [inline src/steps/resource-manager/container-services/converters.ts:32]: Dont throw an error. Wrap the function in try catch. If there is any exception, just log it and return a default value or empty string maybe

File: src/steps/resource-manager/container-services/index.ts [hunk 2]
```diff
@@ -82,6 +84,54 @@ export async function fetchMaintenanceConfigurations(
...
+        async (rolebinding) => {
+          const rolebindingEntity = createRoleBindingEntity(
+            webLinker,
+            rolebinding,
+          );
+          if (!jobState.hasKey(rolebindingEntity._key)) {
+            await jobState.addEntity(rolebindingEntity);
+          }    
+          const relationship = createMappedRelationship({
+            _key: `${rolebindingEntity._key}|IS|ClusterRole:${rolebindingEntity._key}`,
```
Comment (suraj-metron) [inline src/steps/resource-manager/container-services/index.ts:109]: Use a function to return this value

**Review by suraj-metron — CHANGES_REQUESTED** [review]

File: src/steps/resource-manager/container-services/converters.ts (hunk 1 above)
Comment (poornima-metron) [reply src/steps/resource-manager/container-services/converters.ts]: Wrapped the function in try catch

File: src/steps/resource-manager/container-services/index.ts (hunk 2 above)
Comment (poornima-metron) [reply src/steps/resource-manager/container-services/index.ts]: created function to generate key

**Pull request merged**